app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ActiveRecord::Base
+  belongs_to :author, class_name: 'User'
+  belongs_to :posts
+
+  after_save :update_comment_counter
+
+  private
+
+  def update_comment_counter
+    author.increment!(:comments_counter)
+  end
```

Comment:
- [ ] This method should be called when a comment is made to a single post. If you look at your `db/schema.rb` file, you will see that there is a column called `comments_counter` in the `Posts` table. This means that it is the comment_counter that will increase once a comment is made to a post. To reflect this, kindly make it `post.increment!(:comments_counter)` and not `author.increment!(:comments_counter)`. 👍